src/tutorials/Mutable-File-System/08-exercise.md

```diff
@@ -0,0 +1,11 @@
+Move only the files (no directories) from your root directory into the `/some/stuff` directory.
+
+Remember that you can pass an array into `files.mv` as the `from` value. This is useful because it allows you to run the async function only once. Be sure to use the `await` keyword so that the `mv` call completes before the contents of your `/some/stuff` directory are evaluated.
+
+When creating your array to pass in, ensure it only contains files, not directories. Remember that each object in IPFS has a `type` property that you can use to determine whether it's a file or a directory. For files it's the number `0` and for directories it's the number `1`. (Hint: Try the [`filter`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter) or [`forEach`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach) array methods.)
+
+Remember that `files.mv` needs paths, not filenames, so you'll need to prepend each filename in your array with `/`. (Hint: Try the [`map`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) or [`forEach`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach) array methods.)
```

Comment:
I could really do with a reminder of the output of `files.ls` here - I couldn't remember if each object had a `path` or `name` property...